src/sql/util.py

```diff
@@ -157,6 +157,84 @@ def show_deprecation_warning():
     )
 
 
+def check_duplicate_arguments(
+    magic_execute, cmd_from, args, allowed_duplicates
+) -> bool:
+    """
+    Raises UsageError when duplicate arguments are passed to magics.
+    Returns true if no duplicates in arguments or aliases.
+    """
+
+    aliased_arguments = {}
+    unaliased_arguments = []
+
+    for decorator in magic_execute.decorators:
+        assert isinstance(decorator.args, tuple)
+        dec_args = decorator.args
+        if len(dec_args) > 1:
+            aliased_arguments[dec_args[0]] = dec_args[1]
+        else:
+            if dec_args[0].startswith("--") or dec_args[0].startswith("-"):
```

Comment:
in which case will it not start with `--`/`-`

Reply:
The arguments we get from in the `SqlCmdMagic` class decorators don't start with `--` or `-`.

Reply:
what about `--delete`, `--schema` etc? How are these translated?